src/Microsoft.ML.Recommender/MatrixFactorizationTrainer.cs

```diff
@@ -116,6 +127,24 @@ public sealed class Arguments
             [TlcModule.SweepableDiscreteParam("Eta", new object[] { 0.001f, 0.01f, 0.1f })]
             public double Eta = 0.1;
 
+            /// <summary>
+            /// Importance of unobserved (i.e., negative) entries' loss in one-class matrix factorization.
+            /// </summary>
+            [Argument(ArgumentType.AtMostOnce, HelpText = "Importance of unobserved entries' loss in one-class matrix factorization.")]
```

Comment:
>Importance of unobserved entries [](start = 59, length = 32)

1 is super important and 0 is not important at all? Or it's opposite? Can you specify range requirement?
Also I don't see any checks in code for this parameter.

Reply:
Doc extended again. :)